.coafile

```diff
@@ -46,3 +46,7 @@ language = python3
 # 0.8.x (current stable release) needs them.
 ci_keywords, keywords = \#TODO, \# TODO, \#FIXME, \# FIXME
 cs_keywords =
+
+[PythonPackageInit]
+bears = PythonPackageInitBear
```

Comment:
This could go into the `python` section IMO - QuotesBear and SpaceConsistencyBear aren't really related but they're there. Maybe others could chip in too?